environments/pwa.md

```diff
@@ -1,25 +1,33 @@
 ---
 title: "Progressive Web Apps"
-description: Learn how to install the editor PWAs for a browser editing experience closer to native.
+description: Learn how to install the editor PWAs
+for a browser editing experience closer to native.
 ---
 
-In addition to running Coder in a web browser, you can also run it as
-a progressive web application (PWA). Using Coder as a PWA offers you
-an experience that is akin to a native application and offers improved
-performance.
+In addition to running Coder in a web browser, you can also run it as a
```

Comment:
```suggestion
In addition to running Coder in a web browser, you can also run each application as a
```

Probably will fail lint as-is due to line-length, but just to demonstrate wording